### PR TITLE
Edl enhancements

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-8806)

## Description
- Undeprecated the *Legacy Query* parameter.
- Removed *Listen Port* and *credentials* parameters from cortex XSIAM.
- Move out from `Advanced settings` the *query* parameter.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EDL/Integrations/EDL/EDL.yml [hunk 1]
```diff
@@ -17,8 +17,7 @@ configuration:
   display: Indicator Query
   name: indicators_query
   type: 0
-  section: Collect
-  advanced: true
+  advanced: false
```
Comment (DeanArbel) [inline Packs/EDL/Integrations/EDL/EDL.yml:20]: Why remove from Collect?

File: Packs/EDL/Integrations/EDL/EDL.yml [hunk 2]
```diff
@@ -82,6 +83,8 @@ configuration:
   type: 9
   section: Connect
   required: false
+  hidden:
+  - marketplacev2
```
Comment (DeanArbel) [inline Packs/EDL/Integrations/EDL/EDL.yml:87]: Have you tested this?
Last time I checked this wasn't showing as expected.

**Review by DeanArbel — APPROVED** [review]
Looks good. Let's do an offline demo.

File: Packs/EDL/Integrations/EDL/EDL.yml (hunk 1 above)
Comment (adi88d) [reply Packs/EDL/Integrations/EDL/EDL.yml:21]: i reverted it

File: Packs/EDL/Integrations/EDL/EDL.yml (hunk 2 above)
Comment (adi88d) [reply Packs/EDL/Integrations/EDL/EDL.yml:89]: tested in xsoar saas:
image

File: Packs/EDL/ReleaseNotes/3_2_13.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### Generic Export Indicators Service
+
+- Undeprecated the *Legacy Query* parameter.
+- Removed *Listen Port* and *credentials* parameters from cortex XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/ReleaseNotes/3_2_13.md:7]:
```suggestion
- Removed *Listen Port* and *credentials* parameters from Cortex XSIAM.
```

File: Packs/EDL/ReleaseNotes/3_2_13.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+- Move out from `Advanced settings` the *query* parameter.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/ReleaseNotes/3_2_13.md:8]: I am not sure what this means?  Is it no longer part of the Advanced settings but it is part of the regular settings?  Was it just removed?

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

File: Packs/EDL/ReleaseNotes/3_2_13.md (hunk 4 above)
Comment (adi88d) [reply Packs/EDL/ReleaseNotes/3_2_13.md:8]: It means that the `Indicator Query` params was inside the Advanced settings and now we move it out from there to `Collect` section

image

**Pull request merged**